Additional metrics (#920)
imgproxy/imgproxy: docs/prometheus.md

imgproxy can collect metrics for Prometheus. To use this feature, do the following:

1. Set the `IMGPROXY_PROMETHEUS_BIND` environment variable to the address and port that will be listened to by the Prometheus server. Note that you can't bind the main server and Prometheus to the same port.
2. _(optional)_ Set the `IMGPROXY_PROMETHEUS_NAMESPACE` to prepend prefix to the names of metrics, i.e. with `IMGPROXY_PROMETHEUS_NAMESPACE=imgproxy` names will appear like `imgproxy_requests_total`.
3. Collect the metrics from any path on the specified binding.

imgproxy will collect the following metrics:

* `requests_total`: a counter with the total number of HTTP requests imgproxy has processed
* `errors_total`: a counter of the occurred errors separated by type (timeout, downloading, processing)
* `request_duration_seconds`: a histogram of the request latency (in seconds)
* `request_span_duration_seconds`: a histogram of the request latency (in seconds) separated by span (queue, downloading, processing)

> * `requests_in_progress`: a number of requests currently being in progress

Reviewer: ```suggestion
* `requests_in_progress`: the number of requests currently in progress
```

> * `images_in_progress`: a number of images currently being in progress

Reviewer: ```suggestion
* `images_in_progress`: the number of images currently in progress
```
